my-app/src/App.tsx

```diff
@@ -1,26 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
+
 import './App.css';
 
 function App() {
```

Comment:
Since you created `backend` folder, I would recommend you to create a `frontend` folder on the same level and put this `src` folder there, to have a cleaner folder structure

Reply:
tried to make a new folder - frontend and put src there, got _Could not find a required file. Name: index.js_ .
also tried to put in this folder other packages and it didn't work out

Reply:
hm, probably the problem is that you have same package.json for both projects. 
FE should be separated from BE. It can even be in a separate repo. But if you decided to put it in one repo, then make sure BE and FE are independent projects and each has its own package.json